ramls/patron-notice-policy.json

```diff
@@ -207,74 +157,31 @@
             "properties": {
               "sendHow": {
                 "type": "string",
-                "description": "Defines how notice should be sent, before, after or upon",
+                "description": "Defines how notice should be sent, after or upon",
                 "enum": [
                   "Upon At",
-                  "Before",
                   "After"
                 ]
               },
+              "sendWhen": {
+                "type": "string",
+                "description": "Triggering event",
+                "enum": [
+                  "Overdue fine returned",
+                  "Overdue fine renewed"
+                ]
+              },
               "sendBy": {
                 "type": "object",
-                "additionalProperties": false,
-                "description": "Send by options",
-                "properties": {
-                  "duration": {
-                    "type": "integer",
-                    "description": "Duration amount",
-                    "minimum": 1,
-                    "maximum": 100
-                  },
-                  "intervalId": {
-                    "type": "string",
-                    "description": "Interval id",
-                    "enum": [
-                      "Minutes",
-                      "Hours",
-                      "Days",
-                      "Weeks",
-                      "Months",
-                      "Years"
-                    ]
-                  }
-                },
-                "required": [
-                  "duration",
-                  "intervalId"
-                ]
+                "$ref": "interval.json"
               },
               "sendEvery": {
                 "type": "object",
-                "additionalProperties": false,
-                "description": "Send every options",
-                "properties": {
-                  "duration": {
-                    "type": "integer",
-                    "description": "Duration amount",
-                    "minimum": 1,
-                    "maximum": 100
-                  },
-                  "intervalId": {
-                    "type": "string",
-                    "description": "Interval id",
-                    "enum": [
-                      "Minutes",
-                      "Hours",
-                      "Days",
-                      "Weeks",
-                      "Months",
-                      "Years"
-                    ]
-                  }
-                },
-                "required": [
-                  "duration",
-                  "intervalId"
-                ]
+                "$ref": "interval.json"
               }
             },
             "required": [
-              "sendHow"
+              "sendWhen"
```

Comment:
I want to understand this change better. "how" is now not required, but "when" is. At the same time, "when" only has options for "Overdue fine returned" and "Overdue fine renewed". What will happen to other fee/fine notices? Are there other fee/fine notices (f.e. for manual fines)?

Reply:
This is the way it is in other two notice categories in this schema, `loanNotices` and `requestNotices`. This change will not affect any existing notices, since there are none. Manual fees/fines are created in `mod-feesfines` regardless of patron notice policy.

Reply:
Thanks